### DISCUSSION TITLE
Contribution Surveys

### DISCUSSION THREAD

**Discussion by stephen-rowan** [NFT-DAO/Governance-HOLON#96] [Tokenomics]
Hi everyone, I submit for your consideration a design for an NFT-DAO Contribution-Survey based on [QA-DAO work](https://github.com/Quality-Assurance-DAO/DAO-Open-Source/blob/main/Documents/Tokens/04-Contribution-Survey.md).
 
## NFT-DAO-Contribution-Survey

**The key approach is to capture contributions first and assess value last.** 

Why ? Because value led models typically dictate the nature of contributions e.g. a financial incentive contribution model will tend to favor immediate returns, an equal share of rewards upfront will penalize those who have contributed more than others etc.

A contribution led model either tracks or audits DAO member contributions first. Then rewards (in ADA) or recognition (in a Token) are applied to the contribution data afterwards.

## Identity

**This survey encourages DAO members to identify themselves as it provides a record of their contribution.**

However a anon ID option should be available and peer reviewed per work area.

## General Information

This provides an overall summary or context of a member, when they became involved and how many days they have worked.
This could be supplemented with a contribution vote by peers to capture the more intangible contribution a member has made. 

- [ ] Date joined DAO
- [ ] Days / Hours worked overall
- [ ] Cost of contribution (from member perspective)
- [ ] Contribution vote

## Work areas (HOLONs) -

A member can indicate the different work areas or HOLONs they have worked in and provide an estimate of Days / Hours worked in each area.
The areas are Development , Marketing, Governance, Business, Creative, Product and Intangible work.
Intangible work is meant to capture contributions not covered by the scope of a work area.

## Cost calculations (to be decided)

Hours/Days worked - Costs
A variable rate may be applied depending up the cost/rate of the contributor (their hourly rate and work costs).

## Development 

Each work area will be divided into appropriate categories (e.g. Code, UX etc.) and be accompanied with an "Outputs" section where members can record what they have produced, built, documented etc.

- [ ] Code
- [ ] UX
- [ ] Test
- [ ] Support
- [ ] **Outputs** : What has been produced, built, documented etc.

## Marketing 

-  [ ] Campaign
-  [ ] Research
- [ ] **Outputs** : What has been produced, built, documented etc.

## Governance

- [ ] Documentation
- [ ] Organization
- [ ] Procedures
- [ ] **Outputs** : What has been produced, built, documented etc.

## Business

- [ ] Business relationships
- [ ] Negotiation
- [ ] Networking
- [ ] **Outputs** : What has been produced, built, documented etc.

## Product

- [ ] Design
- [ ] Copy
- [ ] Planning
- [ ] **Outputs** : What has been produced, built, documented etc.

## Creative - Art

- [ ] Product Design
- [ ] Video / Film
- [ ] Artwork
- [ ] Art curation
- [ ] Artist outreach
- [ ] **Outputs** : What has been produced, built, documented etc.

## Intangible work

- [ ] Greeting new members
- [ ] Mentoring
- [ ] Social / Emotional Support
- [ ] Communicating news and ideas in Discord
- [ ] **Outputs** : What has been produced, built, documented etc.

# Peer Review

## HOLON

After the survey each contribution is assessed for due diligence by the appropriate work area HOLON - to confirm that there is agreement that the work took place etc.

## DAO

After the survey - intangibles are voted upon per member by the DAO.

# Reward and evaluation

Once the above process is completed a member may choose either to be awarded - 

- [ ] ADA per day / hour = Total *
- [ ] A recognition Token that simply records the above contribution.

*Note - A variable rate may be applied depending up the cost/rate of the contributor (their hourly rate and work costs).

Keeping it simple at this stage allow for the possibility  that a recognition token may become transferable later.

**Comment by newmindflow** [comment]
I would recommend the inclusion of a **Product Holon** with the following contribution categories :

Design
Copy
Planning

**Reply by newmindflow** [reply]
Also, I suggest the inclusion of the following sub-categories to **Creative - Art**:

Art curation
Artist outreach

**Reply by stephen-rowan** [reply]
Thanks @newmindflow - I have added those sub-categories.

What is your view on the upfront "Cost of contribution (from member perspective)". Do you think each member should cost their work ? (e.g. $ per hour). Should this be based on market rates ?

**Reply by newmindflow** [reply]
As at the moment I can't think of a better way to quantify this, it probably should be based on market rates.

**Reply by DanM3rcurius** [reply]
What is meant by "market rates"? Is there anything about the location and the average cost of living implied within "market rates"?

**Reply by newmindflow** [reply]
Good point.
Since Cardano envisions itself as a global infrastructure that aims to offer a level financial playing field, I don't know how much we should rely on geography to determine the rate of pay.
Then again it would be somewhat unfair for people who live in circumstances with a higher cost of living to get paid the same rate as those who live in places with lower average living costs.

In our case for instance we plan on traveling again after Covid ends so we won't be in the same location/country. Making it very hard to estimate the average cost of living.
Or we could recalculate it every month based on where we are, lower for Indonesia higher for the Netherlands for example.

**Reply by stephen-rowan** [reply]
A level playing field does not imply absolute equality. I think the best we can do is ask each member to cost themselves according to their own needs (which vary by circumstance and economy). For due diligence each member should be prepared to justify this cost.

Then their peers need to assess this fairly - recognizing that  (for example) someone living in New York may need to be paid more than someone living in Thailand or that different skills cost more or less etc.

**Comment by DanM3rcurius** [comment]
🤩 @stephen-rowan amazing work 🙏 Will look at this today and propose to employ within next autonomous organisation bootstrap.

**Comment by DanM3rcurius** [comment]
@stephen-rowan: What's the process / workflow to fill this out?
